examples/reference-entity-model/input/json/Cust1.json

```diff
@@ -40,5 +40,22 @@
   "PIN": 2172,
   "Created": "04/03/2017",
   "Status": "New Customer",
-  "SSN": "111-11-1111"
+  "SSN": "111-11-1111",
+  "numbersArray" : [
```

Comment:
I think it would be better to have UI tests for these values rather than adding it to reference project input data. These new data don't have relevance to "Customer" data

Reply:
Thanks, @srinathgit  that's fair. I initially checked with @bsrikan beforehand so I'm curious what he thinks? I could also make it a new input file unrelated to "Customer" data.

Reply:
I think we would like to keep reference project as clean as possible. @bsrikan , let us know your thoughts

Reply:
When we were reviewing the test plan we wanted any enhanced test data to be added to reference project, because it is pushed to HC nightly and most of our engineers and PM use it for reviewing/testing PR.

Reply:
As for tests, Brian has added unit tests for the fix.

Reply:
I agree with @srinathgit, the example reference project should be more like a demo with (Customer) data that ties together (although it is also intended to cover many data use cases).

Reply:
Looks like we're leaning towards taking out this enhancement to test data with certain edge cases to keep the 'Customer' data clean. @rjrudin could you confirm? And I can go ahead and take that out of the PR.

Reply:
@briantang you did add more test data to common.data.ts to cover these cases and I think that's fine. Let's pull those examples out of the reference project for now and push this PR through. I can't see @rjrudin having a problem with this. :-)

Reply:
Thanks @wooldridge, will leave it out for now so we can get this bug fix through.

Reply:
The key to me is that the entity model isn't being used to determine how data in the source panel is displayed - that's not the case, right?

Reply:
@rjrudin that is not the case, display of source data is based on the source data.